Tasks/UniversalPackagesV0/Common/ArtifactToolUtilities.ts

```diff
@@ -1,12 +1,12 @@
 // Placed as a separate file for the purpose of unit testing
+import AdmZip = require('adm-zip');
```

Comment:
I'll approve but you might want to reconsider this dependency due to this [bug](https://github.com/cthackers/adm-zip/issues/112).

Reply:
cool, I will take a look at the alternatives but not block on this.